db/redis.js

```diff
@@ -28,14 +35,21 @@ async function getStatesWindow(locationID, startTime, endTime, windowLength){
     return stateStream
 }
 
+async function clearKeys(){
```

Comment:
@mariocimet I was looking at this and wondering if clearKeys needs some sort of protection, since it deletes all the keys from the redis db? Maybe an extra parameter while calling or something?

Reply:
I think the underlying question here might be - how do we make sure this doesn't delete all our data in production? I think the fact that its only ever called in the test file and the fact that it only clears keys for whatevers on REDIS_CLUSTER_IP_TEST makes that unlikely